src/google/protobuf/compiler/objectivec/objectivec_helpers.h

```diff
@@ -39,10 +39,10 @@
 #include "google/protobuf/descriptor.h"
 #include "google/protobuf/descriptor.pb.h"
 #include "google/protobuf/io/zero_copy_stream.h"
+#include "google/protobuf/compiler/objectivec/names.h"
 
-// clang-format off
```

Comment:
I think you need to keep this wrapping or future clang-format runs might move it.

Reply:
Adding a trailing comment:
// Must be included last.

is how we get around this in other files.  I just updated this for consistency, but it should work the same